### PR TITLE
Add Social Network

### PR DESCRIPTION
**Pull request by everhusk**
Adding new ss58 prefix for social network.

### CODE REVIEW

File: primitives/core/src/crypto.rs [hunk 1]
```diff
@@ -576,6 +576,10 @@ ss58_address_format!(
 		(46, "reserved46", "Reserved for future use (46).")
 	Reserved47 =>
 		(47, "reserved47", "Reserved for future use (47).")
+	SocialTestAccount =>
+		(251, "test-social-network", "Test Social Network, standard account (*25519).")
```
Comment (bkchr) [inline primitives/core/src/crypto.rs:580]: For test nets you can just use the generic substrate prefix.

Comment (everhusk) [reply primitives/core/src/crypto.rs]: Sure no problem, I removed it.

**Review by gavofyork — APPROVED** [review]

**Comment by gavofyork** [comment]
Needs resolving.

**Comment by everhusk** [comment]
Resolved

**Comment by gnunicorn** [comment]
Please point us to your testnet/codebase. Once we verify that your project has reasonable need of a Substrate prefix, we'll be happy to reserve the code.

**Comment by shawntabrizi** [comment]
@gnunicorn their Substrate based application can be found here:

* https://social.network/

Their Substrate project can be found here:

* https://github.com/social-network/blockchain

They are legit.

File: ss58-registry.json [hunk 2]
```diff
@@ -442,7 +442,7 @@
 			"standardAccount": null,
 			"website": null
 		},
-		{
+    {
```
Comment (shawntabrizi) [inline ss58-registry.json:445]:
```suggestion
	{
```

File: ss58-registry.json [hunk 3]
```diff
@@ -442,7 +442,7 @@
 			"standardAccount": null,
 			"website": null
 		},
-		{
+	{
```
Comment (shawntabrizi) [inline ss58-registry.json:445]:
```suggestion
		{
```

File: ss58-registry.json [hunk 4]
```diff
@@ -459,6 +459,15 @@
...
+		},
+		{
+			"prefix": 252,
+			"network": "social-network",
+			"displayName": "Social Network",
+			"symbols": ["NET"],
+			"decimals": [18],
+			"standardAccount": "*25519",
+			"website": "https://social.network"
+    }
```
Comment (shawntabrizi) [inline ss58-registry.json:471]:
```suggestion
		},
		{
			"prefix": 252,
			"network": "social-network",
			"displayName": "Social Network",
			"symbols": ["NET"],
			"decimals": [18],
			"standardAccount": "*25519",
			"website": "https://social.network"
		}
```

**Review by shawntabrizi — APPROVED** [review]

**Comment by shawntabrizi** [comment]
@everhusk please always use tabs when touching substrate stuff :)

**Comment by shawntabrizi** [comment]
bot merge

**Comment by everhusk** [comment]
Thanks Shawn! Fixed the conflict.

**Comment by shawntabrizi** [comment]
bot merge

**Comment by shawntabrizi** [comment]
bot merge

**Pull request merged**